ansible/roles/test/tasks/qos_sai.yml

```diff
@@ -66,12 +66,12 @@
       supervisorctl: state=stopped name=bgpd
       delegate_to: "{{ ansible_host }}_bgp"
     
-    - name: Add iptables rule to drop BGP SYN Packet from peer so that we do not ACK back
-      shell: "iptables -A INPUT -j DROP -p tcp --destination-port bgp"
+    - name: Add iptables rule to drop BGP SYN Packet from peer so that we do not ACK back. Add at top so existing rules don't have precedence over it.
+      shell: "iptables -I INPUT 1 -j DROP -p tcp --destination-port bgp"
```

Comment:
@abdosi can you please change the action in the [pytest](https://github.com/Azure/sonic-mgmt/blob/master/tests/qos/qos_sai_base.py#L380) too? The action should be insert as per Ansible [documentation](https://docs.ansible.com/ansible/2.8/modules/iptables_module.html)

Reply:
@tahmed-dev and @neethajohn  Updated for pytest